ladybug/src/main/java/org/frankframework/ibistesttool/jdbc/ProxyDataSourceFactory.java

```diff
@@ -70,7 +70,7 @@ public void onApplicationEvent(ContextRefreshedEvent event) {
 			delegate = ac.getBean("dataSourceFactory", IDataSourceFactory.class);
 
 			PlatformTransactionManager txManager = ac.getBean("txManager", PlatformTransactionManager.class);
-			SpringUtils.registerSingleton(applicationContext, "ladybugTransactionManager", txManager);
+			SpringUtils.registerSingleton(applicationContext, "ladybugTransactionManager", txManager); //Should use this instead of the Ladybug's OptionalJtaTransactionManager
```

Comment:
Maar is het gegarandeerd, dat deze nu wordt gebruikt?

Reply:
Ja dat heb ik getest, de xml heeft prioriteit over de `@Configuration` annotaties.